signals.go

```diff
@@ -0,0 +1,30 @@
+package main
```

Comment:
Thought about that. I want to get this out the door, but I have a TODO to make this more general and it's own package to share with envconsul and consul-replicate in the future.